x-pack/test/functional_with_es_ssl/apps/cases/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ loadTestFile }: FtrProviderContext) => {
+  describe('Cases stack management', function () {
```

Comment:
can you explain me a bit more how are these ci group tags used? how do I decide which one to use?

Reply:
The tests are run in parallel tracks on CI in order to keep execution time acceptable for PRs. The assignment to tracks is done in code via the `ciGroupXY` tag in test suites. If no such tag exists in the `describe` hierarchy of a test, it will not be picked up by CI.
As a rule of thumb for new tests and track assignments, we look at a recent main build (e.g. [here](https://buildkite.com/elastic/kibana-hourly/builds/13072), check the group execution times and pick the group that has the lowest run time for the new suite. For tests in the `x-pack/test` directory we're looking at `Default CI Group`, for everything in the root `test` directory, it would be the `OSS CI Group`.
Note, that we tend to put the CI group labels rather high up in the suite hierarchy (e.g. on `apps` level for functional tests) and only break it down if a block becomes too big (this is usually done by the ops team). [Here's an example](https://github.com/elastic/kibana/blob/main/x-pack/test/functional/apps/ml/index.ts#L40) how groups within on app are broken down into multiple groups.

Reply:
Thanks for the explanation. I added the tag as you suggested.